src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/arm/conv_fq.cpp

```diff
@@ -121,19 +123,33 @@ TEST_P(ConvAndFQ, CompareWithRefs) {
 
 namespace {
 
-std::vector<InputShape> inputShapes{{{}, {{4, 3, 2, 2}}},
+std::vector<InputShape> inputShapes{{{}, {{1, 3, 2, 2}}},
                                     {{-1, 3, -1, 2}, {{1, 3, 4, 2}}}};
 
-std::vector<std::vector<std::vector<float>>> quantizeIntervals{
+std::vector<std::vector<std::vector<float>>> perTensorQuantizeIntervals{
     {{-1.28f}, {1.27f}, {-1.28f}, {1.27f}},
     {{0.f}, {2.55f}, {0.f}, {2.55f}},
 };
 
-INSTANTIATE_TEST_SUITE_P(smoke_ConvAndFQ_CPU,
+std::vector<std::vector<std::vector<float>>> perChannelQuantizeIntervals{
+    {{-1.28f, -1.0f, -0.8f}, {1.27f, 1.0f, 0.8f}, {-1.28f, -1.0f, -0.8f}, {1.27f, 1.0f, 0.8f}},
+};
+
+INSTANTIATE_TEST_SUITE_P(smoke_ConvAndFQ_CPU_PerTensor,
```

Comment:
Can we check somehow that in case of PerTensor dequantization we execute the Convolution in low precision, and in case of PerChannel - in f32? I'd propose introducing the check similar to the `MatmulWeightsDecompression::check_results()` (can be found in src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/classes/matmul_weights_decompression.cpp):
1. Find convolution layer by `ov::exec_model_info::LAYER_TYPE`
2. Check that `ov::exec_model_info::RUNTIME_PRECISION` is expected